src/poliastro/tests/test_bodies.py

```diff
@@ -33,6 +37,42 @@ def test_body_printing_has_name_and_symbol():
 
 
 def test_earth_has_k_given_in_literature():
-    expected_k = 3.986e14 * u.m ** 3 / u.s ** 2
+    expected_k = 398600.44 * u.km ** 3 / u.s ** 2
     k = bodies.Earth.k
     assert_quantity_allclose(k.decompose([u.km, u.s]), expected_k)
+
+def test_soi_calculation_working():
+    _name = 'Planet'
+    _k = 1 * u.km**3 / u.s**2
+    _parent_name = 'Start'
+    _parent_k = 32* _k
+    _parent = bodies.Body.from_parameters( _parent_k, _parent_name, _parent_name,
+                                         1 * u.km, None, None)
+    _orbit = Orbit.circular(_parent, 3 * u.km)
+    _planet = bodies.Body.from_parameters(_k, _name, _name,
+                                          0.1 * u.km, _parent, _orbit)
+    _planet.calculate_soi()
+    _soi = _planet.soi
+    expected_soi = 1 * u.km
+    assert_quantity_allclose(_soi.to(u.km), expected_soi)
+    
+def test_generalized_Titius_Bode_Law():
+    _bodies = ['Mercury', 'Venus', 'Earth', 'Mars', 'Ceres',
+               'Jupiter', 'Saturn', 'Uranus', 'Neptune']
+    au = 149597870.700 * u.km 
+    expected_approx = au * 0.2075 * 1.7327 ** arange(1,10)
+    obtained = []
+    for _name in _bodies:
```

Comment:
I would advise keeping the cyclomatic complexity of the tests as unity and instead parametrize them. See for example:

https://github.com/poliastro/poliastro/blob/v0.6.0/src/poliastro/tests/test_iod.py